docs/config.md

```diff
@@ -131,3 +135,27 @@ The JSON file has the following fields:
   * `jira`: Optional Jira server and login information
   * `whitesource`: Optional whitesource server authentication information
   * `github_oauth`: Optional project specific GitHub OAuth token
+
+### create findings-[project].yaml files for each project
+
+The `findings-[project].yaml` file (where `[project]` is replaced with the project name used in the config file) contains information used in the generation of the HTML report.  The file contains a list of `findings` with the following fields:
+* `id` Id unique to each finding
+* `priority` Priority of the finding as displayed in the report
+* `matches-license` license name that this finding applies to as it appears in the fossology
+* `matches-path` optional file path that the finding applies to
```

Comment:
There is also `matches-subproject` -- I _think_ that that will:
* if included on its own, flag a single instance of this finding for that subproject's report but without tying it to any particular files
* if included together with a `matches-path` or `matches-license` option, require an `AND` of both of them in order to flag an instance of this finding

I'm not 100% certain but I _think_ that's how it behaves...

Reply:
I added the matches-subproject and also restructured this section a bit to group all the matches fields together.